release/ray_release/glue.py

```diff
@@ -167,6 +167,8 @@ def run_release_test(
         raise ReleaseTestSetupError(f"Error setting up release test: {e}") from e
 
     pipeline_exception = None
+    # non critical for some tests. So separate it from the general one.
+    fetch_command_exception = None
```

Comment:
I don't think we need to re-raise this. Let's just have an well phrased error message when the results are empty

Reply:
Talked offline, we will stick with ResultError instead of introducing a new one.